test/test_retriever.py

```diff
@@ -246,107 +244,122 @@ def test_sort_csv():
 
 def test_is_empty_null_string():
     """Test for null string"""
-    assert(is_empty("") == True)
+    assert (is_empty("") == True)
 
 
 def test_is_empty_empty_list():
     """Test for empty list"""
-    assert(is_empty([]) == True)
+    assert (is_empty([]) == True)
 
 
 def test_is_empty_not_null_string():
     """Test for non-null string"""
-    assert(is_empty("not empty") == False)
+    assert (is_empty("not empty") == False)
 
 
 def test_is_empty_not_empty_list():
     """Test for not empty list"""
-    assert(is_empty(["not empty"]) == False)
+    assert (is_empty(["not empty"]) == False)
 
 
 def test_clean_input_empty_input_ignore_empty(monkeypatch):
     """Test with empty input ignored"""
+
     def mock_input(prompt):
         return ""
+
     monkeypatch.setattr('retriever.lib.datapackage.input', mock_input)
-    assert(clean_input("", ignore_empty=True) == "")
+    assert (clean_input("", ignore_empty=True) == "")
 
 
 def test_clean_input_empty_input_not_ignore_empty(monkeypatch):
     """Test with empty input not ignored"""
+
     def mock_input(prompt):
         mock_input.counter += 1
         if mock_input.counter <= 1:
             return ""
         else:
             return "not empty"
+
     mock_input.counter = 0
     monkeypatch.setattr('retriever.lib.datapackage.input', mock_input)
-    assert(clean_input("", ignore_empty=False) == "not empty")
+    assert (clean_input("", ignore_empty=False) == "not empty")
```

Comment:
The extra parentheses should be removed here in the same way you've been removing them further down.